src/manager.cc

```diff
@@ -334,8 +336,36 @@ fastcat::Manager::GetDeviceStatePointers()
 }
 
 double fastcat::Manager::GetTargetLoopRate() { return target_loop_rate_hz_; }
+
 bool   fastcat::Manager::IsFaulted() { return faulted_; }
 
+void fastcat::Manager::InitializeActuatorNames() {
+  actuator_names_.clear();
+  for (auto& device : jsd_device_list_) {
+    if(device->GetState()->type == ACTUATOR_STATE) {
+      actuator_names_.push_back(device->GetName()); 
+    }
+  }
+}
+
+const std::vector<std::string>& fastcat::Manager::GetActuatorNames() {
+  return actuator_names_;
+}
```

Comment:
It would be more general and be more future proof for this to be of the form:
``` c++
void fastcat::Manager::GetDeviceNamesByType(std::vector<std::string>& name_list, DeviceStateType type) {
  name_list.clear();
  // for all devices
      // if device type matches type
          name_list.push_back(name)
    return name_list;
}

Reply:
This way we don't need to manage the memory and don't need `InitializeActuatorNames()` either